scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +65,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
+        """
+        if metric_type == 'string':
+            return val
+
+        try:
+            return json.loads(val)
```

Comment:
Tested!!!!! Against my own local zabbix (with the help of @joelddiaz and @twiest )

float: docker.storage.data.space.available
int: docker.container.existing.dns.resolution.failed
string: kernel.uname.distro

All viewed as sent on the web.
@twiest ok to merge?